editor/property_editor.cpp

```diff
@@ -376,7 +376,7 @@ bool CustomPropertyEditor::edit(Object *p_owner, const String &p_name, Variant::
 			if (hint == PROPERTY_HINT_RANGE) {
 
 				int c = hint_text.get_slice_count(",");
-				float min = 0, max = 100, step = 1;
+				float min = 0, max = 100, step = .01;
```

Comment:
to be honest this part didn't seem to affect the property editor, leaving at 1 still allowed the property to have float values in the range. Though I probably am missing how it's used, which is why I changed it anyway.

Reply:
This is for the spinbox (used when dragging up/down).